README.adoc

```diff
@@ -39,15 +39,18 @@ CRUD operations, JPA also reduces the burden of having to write JDBC and SQL cod
 database operations and takes care of database vendor specific differences. This allows you to 
 focus on the business logic of your application instead of wasting time implementing repetitive CRUD logic.
 
-The application that you will be working with is the event manager, which is composed of a UI and 
+The application that you will be working with is an event manager, which is composed of a UI and 
 an event microservice for creating, retrieving, updating, and deleting events. The application uses an 
 Embedded Derby database as a datastore for all the events.
 
 You will use JPA annotations to define an entity class whose fields are persisted to the 
 database. The interaction between your application and the database is mediated by the persistence 
-context that is managed by an EntityManager. In this guide you will learn how to use a container-managed 
-EntityManager that will be obtained by the application through injection. This means, that the lifecycle
-of the EntityManager will be managed transparently by the Java EE container.
+context that is managed by an EntityManager. The EntityManager can be application-managed or
+container-managed. In this guide you will be using a container-managed EntityManager, placing the
+burden of managing the lifecycle of the EntityManager on the application server. The lifetime of the
```

Comment:
"lifetime" => I think you meant "lifecycle"? 

Btw, these description with each concept broke down into small sentences sounds much better!